server/src-lib/Hasura/Logging.hs

```diff
@@ -108,6 +108,7 @@ data InternalLogTypes
  | ILTJwkRefreshLog
  | ILTTelemetry
  | ILTSchemaSyncThread
+ | ILTPollerLog
```

Comment:
Shouldn't be present, probably a bad merge somewhere up the chain.

Reply:
So remove ILTPollerLog references? There's also a case-match on line 126.